.librarian/state.yaml

```diff
@@ -0,0 +1,10 @@
+image: us-central1-docker.pkg.dev/cloud-sdk-librarian-prod/images-prod/python-librarian-generator@sha256:c8612d3fffb3f6a32353b2d1abd16b61e87811866f7ec9d65b59b02eb452a620
```

Comment:
When should we update this SHA? How? Maybe include a comment here or in a README.